isotovideo

```diff
@@ -220,7 +189,7 @@ load_test_schedule;
 
 # start the command fork before we get into the backend, the command child
 # is not supposed to talk to the backend directly
-($cmd_srv_process, $cmd_srv_fd) = commands::start_server($cmd_srv_port = $bmwqemu::vars{QEMUPORT} + 1);
+my ($cmd_srv_process, $cmd_srv_fd) = commands::start_server(my $cmd_srv_port = $bmwqemu::vars{QEMUPORT} + 1);
```

Comment:
I didn't know it was possible to have to `my` keywords in the same statement.

Reply:
We use it all the times when not using signatures, e.g.:

```perl
my function {
    my ($param1, $param2, %all_the_rest) = @_;
    …
```

Reply:
But in this statement the `my` keyword is only used once (the `my function` line doesn't count into the statement).

Reply:
It's not very common, since it tends to make the code less readable.

Reply:
> We use it all the times when not using signatures, e.g.:
> 
> ```perl
> my function {
>     my ($param1, $param2, %all_the_rest) = @_;
>     …
> ```

where do we use `my function`? somewhere in openqa or os-autoinst?